go.mod

```diff
@@ -45,6 +45,7 @@ require (
 require (
```

Comment:
Run `go mod tidy` since some deps are unused.

Reply:
fixed in fixed in fee1a31da521fd02821b509be37e72d0c975537c